docs/one-wire-devices.md

```diff
@@ -0,0 +1,150 @@
+---
+title: One-Wire Devices
+---
+
+One-wire devices including but not limited to temperature loggers, battery monitors, humidity sensors, voltage and current sensors can be used with a wide range of controllers and processors in the Omega2 family like Omega2, Omega2+, Omega2S, and more through One-wire communication protocol. 
+
+The following is the scope of this documentation:
+
+<ul>
```

Comment:
No need to list out the contents manually, docusaurus will automatically generate a table of contents and display it on the right side. 
Here's an example from the hardware pwm article:
<img width="1440" alt="image" src="https://github.com/OnionIoT/documentation/assets/6757320/623023de-3b01-4310-b585-7e944d8f4a65">

This is based on the markdown headings. See next comment for details